src/libraries/System.Private.CoreLib/src/System/Char.cs

```diff
@@ -510,12 +500,12 @@ public static bool IsLetter(string s, int index)
                 throw new ArgumentOutOfRangeException(nameof(index));
             }
             char c = s[index];
-            if (IsLatin1(c))
+            if (IsAscii(c))
```

Comment:
Shouldn't this stay `IsLatin1` instead of `IsAscii`? It handles Latin-1 characters like 'À' (U+00C0) correctly.

Reply:
Latin-1 now have `\u00aa` and `\u00ba` of OtherLetter category. So the check 


```C#
return (Latin1CharInfo[c] & (IsUpperCaseLetterFlag | IsLowerCaseLetterFlag)) != 0;
```

is not correct any more for Latin-1 but it is still correct for ASCII.


CharUnicodeInfo.GetUnicodeCategoryInternal still handle \u00C0 correctly. 

---
In reply to: [475768625](https://github.com/dotnet/runtime/pull/41200#discussion_r475768625) [](ancestors = 475768625)